src/app/info/components/ChartGovernanceActiveDelegates.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useEffect, useRef, useState } from "react";
+import {
+  Area,
+  CartesianGrid,
+  ComposedChart,
+  Line,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import ChartTabs from "./ChartTabs";
+import useTenantColorScheme from "@/hooks/useTenantColorScheme";
+import type { MetricTimeSeriesValue } from "@/lib/types";
+import { FREQUENCY_FILTERS } from "@/lib/constants";
+
+interface Props {
+  getData: (
+    metric: string,
+    frequency: string
+  ) => Promise<{ result: MetricTimeSeriesValue[] }>;
+}
```

Comment:
is there a reason for passing the `getData` function as a prop vs just defining it here? I already feel like the data fetching function is tightly coupled to this component anyways since it requires metric and frequency. So my first thought if passing the data fetcher down so the chart is abstracted doesn't really stick as much here.

More of a nit / question though, not a deal breaker.

Reply:
@mcgingras getData is a server function that get's passed to the client component. I guess that's the only reason. I agree that having the data method inside the chart would be a better design.

Reply:
ah okay, thanks!